compose/docker-compose.yml

```diff
@@ -57,6 +57,16 @@ services:
       STORM_WEBDAV_AUTHZ_SERVER_ENABLE: true
       STORM_WEBDAV_AUTHZ_SERVER_ISSUER: https://storm.test.example:8443
       STORM_WEBDAV_JVM_OPTS: -Dspring.profiles.active=policies,issuers
+      OTEL_EXPORTER_OTLP_ENDPOINT: https://otello.cloud.cnaf.infn.it/collector
```

Comment:
Maybe we should put this variable in the github action configuration (and use https://otello.cloud.cnaf.infn.it:8443/collector)